dist/common/scripts/scylla_io_setup

```diff
@@ -30,6 +30,60 @@ import logging
 import sys
 import scylla_blocktune as blocktune
 
+def run_iotune():
```

Comment:
When you send patches to scylla-dev make sure to put this change in a separate patch.